client/src/pages/room.jsx

```diff
@@ -0,0 +1,252 @@
+import { useContext, useEffect, useState } from 'react';
+import { Context } from '../context/context.jsx';
+import { Formik, Form, Field } from 'formik';
+import { userService } from '../services/userService.js';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ChatBlock } from './ChatsBlock.jsx';
+
+function messageValidation(newMessage) {
+  if (newMessage.trim().length > 300) {
+    return 'Message is too long';
+  }
+}
```

Comment:
The `messageValidation` function does not return anything when the message is valid. It would be better to explicitly return `undefined` or `null` at the end of the function to indicate that there are no validation errors.